CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Changelog
 
 ### 4.0.0 (Next)
+* [#176](https://github.com/alexa-js/alexa-app/pull/176): Added flexibility to attach alexa-app to express app or router - [@bgjehu](https://github.com/bgjehu).
```

Comment:
> Maybe we should raise an exception if both app and router were specified?

@dblock If that is to be the case, then maybe we should simplify it to one option, something between `expressApp` and `router`.